src/main/webapp/app/course/viewCourse.xhtml

```diff
@@ -0,0 +1,27 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:h="http://java.sun.com/jsf/html">
+<h:head>
+	<title>Utenti salvati</title>
+</h:head>
+<h:body>
+	<h3>View of #{courseController.selectedCourse.name} course</h3>
+	<h:form id="formViewCourse">
+		<h:panelGrid id="panelViewCourse" columns="2" border="0">
+			<h:outputLabel value="#{msg['id']}"/>
+			<h:outputText value="#{courseController.selectedCourse.id}" />
+			
+			<h:outputLabel value="#{msg['name']}"/>
+			<h:outputText value="#{courseController.selectedCourse.name}" />
+			
+			<h:outputLabel value="#{msg['description']}"/>
+			<h:outputText value="#{courseController.selectedCourse.description}" />
+			
+			<h:outputLabel value="#{msg['enabled']}"/>
+			<h:selectBooleanCheckbox value="#{courseController.selectedCourse.enabled}" disabled="true"/>
+			
+			<h:commandButton id="back-button" value="Back" action="homeCourse.xhtml" />
```

Comment:
words Back and Home should be in the property file. Same for all buttons in the app.